civiform_config.example.sh

```diff
@@ -251,3 +262,8 @@ export APPLICANT_OIDC_EMAIL_ATTRIBUTE="email"
 export APPLICANT_OIDC_FIRST_NAME_ATTRIBUTE=""
 export APPLICANT_OIDC_MIDDLE_NAME_ATTRIBUTE=""
 export APPLICANT_OIDC_LAST_NAME_ATTRIBUTE=""
+
+# REQUIRED
+# The discovery metadata URI provideded by the ADFS provider.
+# Usually ends in .well-known/openid-configuration
+export ADFS_DISCOVERY_URI="https://civiform-staging.us.auth0.com/.well-known/openid-configuration"
```

Comment:
This isn't a ADFS url - It's probably a bad example

Reply:
How about this one:
https://login.microsoftonline.com/common/v2.0/.well-known/openid-configuration

From
https://docs.microsoft.com/en-us/azure/active-directory/develop/userinfo

Reply:
Thanks! done.